logpyle/__init__.py

```diff
@@ -877,8 +881,6 @@ def set_constant(self, name: str, value: Any) -> None:
             self.db_conn.execute("insert into constants values (?,?)",
                     (name, value))
 
-        self._commit()
```

Comment:
Is it wise to remove this?

Reply:
I'm not sure. Due to the high value of `commit_interval`, neither of these cases (this one and the one in add_internal) got ever hit in any of our runs. Maybe it would be better to run `save`  at the beginning of the first time step or so?

Reply:
> Maybe it would be better to run `save` at the beginning of the first time step or so?

I implemented this in e962912.